server/server.js

```diff
@@ -90,5 +90,45 @@ module.exports = function(port, db, githubAuthoriser) {
         });
     });
 
+    app.post("/api/messages", function(req, res) {
+        var senderID = req.session.user;
+        var conversationID = req.body.conversationID;
+        var message = req.body.contents;
+        if (message === "") {
+            res.sendStatus(201);
+            return;
+        }
+        conversations.findOne({
```

Comment:
Fine for now, but might be worth considering how to use promises instead of callbacks here.